### DISCUSSION TITLE
Em muốn đổi giao diện về tiếng việt thì làm sao ạ?

### DISCUSSION THREAD

**Discussion by nonqxt86** [vinadevs/VinaText#13] [Q&A]
Em không biết phải chỉnh ở chỗ nào. Nhờ bác chỉ giúp ạ. Em cảm ơn.

**Comment by vinadevs** [comment]
Em vẫn đang trong quá trình dịch app sang tiếng việt bác. Vì nhiều chức năng quá nên dịch hơi lâu chút.
Bác chịu khó đợi thêm chút thời gian. :D
